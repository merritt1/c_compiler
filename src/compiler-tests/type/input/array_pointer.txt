int main(void) {
signed int a[10], *p;
p = (signed int *)a;
}
