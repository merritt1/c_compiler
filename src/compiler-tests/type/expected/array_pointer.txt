=============== PARSE TREE ===============
signed int (main(void)) {
signed int (a[10]), (*(p));
(p=(signed int(*)) a);
}
================= SYMBOLS ================
Symbol Table ID: 0
  main -> function(signed int, [void])

Symbol Table ID: 1
  a -> array(10, signed int)
  p -> pointer(signed int)

